features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Login/Login.js

```diff
@@ -69,9 +70,18 @@ class Login extends Component {
             // Update environment to discard default environment configuration
             const environment = environments[environmentId];
             Utils.setEnvironment(environment);
+
+            // Set authentication status of environments
+            this.setLoginStatusOfEnvironments(environments);
         });
     }
 
+    setLoginStatusOfEnvironments(environments) {
+        this.state.loginStatusEnvironments = environments.map(
```

Comment:
Hi @RenukaFernando , Any reason for manipulating the state object directly? if not Please [do not modify the State directly](https://reactjs.org/docs/state-and-lifecycle.html#do-not-modify-state-directly), This could lead to unexpected results.

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/4984/commits/95f88fd02481997baad4d96e55a9ccc515c1dd2d